### PR TITLE
Provide a way to return config from the test database

### PR DESCRIPTION
**Pull request by sethvargo**
Going to use this in some upcoming stuff, but here's an easier PR to review 😄

### CODE REVIEW

File: internal/verification/database/health_authority_db_test.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func TestAddRetrieveHealthAuthorityKeys(t *testing.T) {
 	}
 
 	wantKeys := []*model.HealthAuthorityKey{
-		&model.HealthAuthorityKey{
+		{
```
Comment (mikehelmick) [inline internal/verification/database/health_authority_db_test.go:80]: covered in https://github.com/google/exposure-notifications-server/pull/502

Comment (sethvargo) [reply internal/verification/database/health_authority_db_test.go]: Done

**Pull request merged**